pkgs/development/python-modules/protobuf3-to-dict/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchPypi, protobuf, six }:
+
+buildPythonPackage rec {
+  pname = "protobuf3-to-dict";
+  version = "0.1.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0nibblvj3n20zvq6d73zalbjqjby0w8ji5mim7inhn7vb9dw4hhy";
+  };
+
+  doCheck = false;
+
+  propagatedBuildInputs = [ protobuf six ];
+
+  meta = with lib; {
+    description =
+      "A teeny Python library for creating Python dicts from protocol buffers and the reverse";
```

Comment:
```suggestion
    description = "A teeny Python library for creating Python dicts from protocol buffers and the reverse";
```

Reply:
What do you use to format, @SuperSandro2000 ? `nixfmt` put this line break :)

Reply:
nixpkgs-fmt which does less line breaks like this which is annoying when doing treewide changes.